sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -2308,6 +2309,18 @@ class Dataset[T] private[sql](
   }
 
   /**
+   * Get the Dataset's current storage level, or StorageLevel.NONE if not persisted.
+   *
+   * @group basic
+   * @since 2.0.0
+   */
+  def getStorageLevel(): StorageLevel = {
```

Comment:
I thought it would be more familiar for RDD users, but no strong feeling about it. Will update